setup.py

```diff
@@ -70,7 +69,10 @@ def get_packages(package):
         'importlib_metadata>=3.10',
         'packaging>=20.5'
     ],
-    extras_require={"i18n": ['babel>=2.9.0']},
+    extras_require={
+        "i18n": ['babel>=2.9.0'],
+        'lunrpy': ['lunr']
```

Comment:
I considered that. However, `lunr` is also a JavaScript lib and I wanted to be clear we are referring the Python lib here. Maybe that's not relevant because we are using `pip` so of course it is the Python lib.

Reply:
I would go for `lunr` for consistency with the package name.